src/tools/pub/pubspec.md

```diff
@@ -395,9 +400,41 @@ the pattern is considered relative to the package's root directory.
 {{site.alert.end}}
 
 {{site.alert.version-note}}
-  Support for the `false_secrets` field was added in Dart 2.15.
+  Dart 2.15 added support for the `false_secrets` field.
 {{site.alert.end}}
 
+### Screenshots
+
+Packages can showcase their widgets or other visual elements using screenshots
+displayed on their package page.
+
+A package can list up to 10 screenshots under the `screenshots` field.
+Each screenshot includes one `description` and one `path`. 
+The `description` explains what the screenshot depicts in no more than 160 
+characters. 
+For example:
+
+```yaml
+screenshots:
+  - description: 'This screenshot shows the transformation of a number of bytes 
+  to a human-readable expression.'
+    path: path/to/image/in/package/500x500.webp
+  - description: 'This screenshot shows a stack trace returning a human-readable
+  representation.'
+    path: path/to/image/in/package.png
+```
+
+Screenshots are limited to:
+- File size: max 4 MB per image.
+- File types: `png`, `jpg`, `gif`, or `webp`. 
+- Static and animated images are both allowed.
+
+It is recommended to keep screenshot files small, since the files will be 
+included in each download of the package.
```

Comment:
**suggestion:** We can express our opinion here as the way to go. No need to couch your language.
```suggestion
Keep screenshot files small. Each download of the package includes all screenshot files.
```

Reply:
Done.